src/Core/Tool/Authorizer/ExportJobAuthorizer.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ * Ushahidi Export Job Authorizer
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Application
+ * @copyright  2018 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+namespace Ushahidi\Core\Tool\Authorizer;
+
+use Ushahidi\Core\Entity;
+use Ushahidi\Core\Tool\Authorizer;
+use Ushahidi\Core\Traits\AdminAccess;
+use Ushahidi\Core\Traits\OwnerAccess;
+use Ushahidi\Core\Traits\UserContext;
+use Ushahidi\Core\Entity\Permission;
+use Ushahidi\Core\Traits\PrivAccess;
+use Ushahidi\Core\Traits\PrivateDeployment;
+use Ushahidi\Core\Tool\Permissions\AclTrait;
+
+class ExportJobAuthorizer implements Authorizer
+{
+	// The access checks are run under the context of a specific user
```

Comment:
Love the comments about why we have each auth trait here , even more after the user issue